glusterd2/xlator/global.go

```diff
@@ -15,6 +20,15 @@ var (
 // Load load all available xlators and intializes the xlators and options maps
 func Load() (err error) {
 
+	defer func() {
+		if r := recover(); r != nil {
+			log.Info(string(debug.Stack()))
+			err = fmt.Errorf("recover()ed at xlator.Load(): %s", r)
+			log.Error("Your version of glusterfs is incomaptible. ",
+				"Please install latest glusterfs from source (branch: master)")
```

Comment:
We need to change this again once 4.1 releases. Can we change this message as "Please install latest glusterfs (>4.0.0)"

Reply:
A [dependent change](https://review.gluster.org/#/c/19466/) from glusterfs that adds options levels (basic, advanced etc) isn't present in 4.0 release.

Reply:
Ok, we can change this again during 4.1 release